exercises/bank-account/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Your task is to simulate a bank supporting opening/closing accounts, withdrawals, and deposits of money.
+
+As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), make sure that your bank accounts can safely be accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
```

Comment:
Is it worth being a bit more verbose here and spelling out what needs doing in more detail?

Reply:
Maybe so. Thoughts on wording?

Reply:
```suggestion
As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), your bank software must allow accounts to be safely accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
For example, there may be many deposits and withdrawals occurring in parallel; you need to ensure there is no [race conditions][helpful link] between when you read the account balance and set the new balance.
```